torch/_subclasses/fake_tensor.py

```diff
@@ -965,6 +977,163 @@ def should_allow_numbers_as_tensors(func: OpOverload):
     )
 
 
+_DISPATCH_CACHE = {}
+
+
+class _Boolean(NamedTuple):
+    value: bool
+
+
+_True = _Boolean(True)
+_False = _Boolean(False)
+
+
+class FakeTensorMetadata(NamedTuple):
```

Comment:
There are a lot of other fields that can be set here: https://github.com/pytorch/pytorch/blob/main/torch/_subclasses/meta_utils.py#L45-L75

Reply:
For instance, storage_offset is not being included